tap_github/repository_streams.py

```diff
@@ -130,6 +132,14 @@ def validate_response(self, response: requests.Response) -> None:
                         "Removing it from list"
                     )
                     continue
+                # check if repo has moved or been renamed
+                if repo_full_name.lower() != name_with_owner.lower():
+                    # the repo name has changed, log some details, and move on.
+                    self.logger.info(
+                        f"Repository name changed: {repo_full_name} \t"
+                        f"New name: {name_with_owner}"
+                    )
```

Comment:
Nit: Use printf-style for logs

```suggestion
                    self.logger.info(
                        "Repository name changed: %s \tNew name: %s",
                        repo_full_name,
                        name_with_owner,
                    )
```

https://blog.pilosus.org/posts/2020/01/24/python-f-strings-in-logging/

Reply:
I feel like f-strings are a bit more readable in this case no? If Meltano has a strong feeling for `%s` I'll open an issue to change other instances of logs as well